Dockerfile

```diff
@@ -45,6 +45,8 @@ ARG DOWNLOAD_URL
 ARG DOWNLOAD_USER
 # allow to override the list of addons to package by default
 ARG ADDONS="exo-chat:1.4.2 exo-tasks:1.2.3 exo-remote-edit:1.2.2"
+# Default base directory on the plf archive
+ARG ARCHIVE_BASE_DIR=platform-${EXO_VERSION}
```

Comment:
Yes you are right, it's fixed.

On the previous PR, I used '*'
```
mv /srv/downloads/* ${EXO_APP_DIR} && \
```
it was working but it was not really clean.